community/cypher/cypher-planner-3.4/src/test/scala/org/neo4j/cypher/internal/compiler/v3_4/ast/rewriters/ExpandStarTest.scala

```diff
@@ -60,77 +60,6 @@ class ExpandStarTest extends CypherFunSuite with AstConstructionTestSupport {
       "match p=(a:Start)-->(b) return a, b, p")
   }
 
-  test("rewrites * in RETURN GRAPHS") {
-    assertMultipleGraphsRewrite(
-      "WITH 1 AS a GRAPH AT 'url' AS foo, GRAPH AT 'url2' AS bar RETURN * GRAPHS *",
```

Comment:
Do I understand correct that this syntax has been removed from the CIP? Or is there another reason theses tests were deleted?

Reply:
That's correctly understood.